src/hud.js

```diff
@@ -56,16 +56,23 @@ class HUD {
         // we start at 25.
         this.hydration = 25;
         this.nutrition = 25;
+        // Clock starts at zero
+        this.clock = {
+          'hours': 0,
```

Comment:
You created the members of this object as strings 'hours', etc. but reference it as `clock.hours`. Weirdly enough, this is technically valid JavaScript, but it's a bit naughty. Could you change this to just `hours`, etc.